ddtrace/tracer.py

```diff
@@ -933,8 +933,7 @@ def execute():
         """
 
         def wrap_decorator(f: AnyCallable) -> AnyCallable:
-            # FIXME[matt] include the class name for methods.
-            span_name = name if name else "%s.%s" % (f.__module__, f.__name__)
+            span_name = name if name else "%s.%s" % (f.__module__, f.__qualname__)
```

Comment:
Thanks for the contribution. 

Is the motivation for this change just to address the FIXME comment or would you like to have this default name on spans generated by `tracer.wrap(...)`. 

Updating the default names of spans could break monitors, dashboards, and metrics for many datadog customers. Instead of  
```suggestion
            span_name = name if name else "%s.%s" % (f.__module__, f.__name__)
```